ext/hal/microchip/mec/mec1501/component/espi_vw.h

```diff
@@ -420,7 +420,7 @@ inline void
 mec_espi_msvw_irq_sel_set(ESPI_MSVW_REG * p, enum espi_msvw_src src,
 			  enum espi_msvw_irq_sel isel)
 {
-	volatile uint8_t *psrc = (volatile uint8_t *)p->SRC_IRQ_SEL;
+	volatile uint8_t *psrc = (volatile uint8_t *)&p->SRC_IRQ_SEL;
```

Comment:
Is this change required?

Reply:
Yes, this is needed to properly enabled interrupt though it's convoluted
@scottwcpg can we clean this up this way  
       p->SRC_IRQ_SEL = (isel << src);